resources/js/beatmapsets-show/scoreboard/main.tsx

```diff
@@ -6,26 +6,28 @@ import BeatmapExtendedJson from 'interfaces/beatmap-extended-json';
 import { route } from 'laroute';
 import { computed, makeObservable, observable } from 'mobx';
 import { observer } from 'mobx-react';
+import modNames from 'mod-names.json';
+import core from 'osu-core-singleton';
 import * as React from 'react';
 import { classWithModifiers } from 'utils/css';
 import { trans } from 'utils/lang';
 import { SwitchError } from 'utils/switch-never';
+import ModJson from '../../interfaces/mod-json';
```

Comment:
(also will need to be moved)

```suggestion
import ModJson from 'interfaces/mod-json';
```